apps/services/user-notification/src/app/modules/notifications/notifications.controller.ts

```diff
@@ -91,6 +96,9 @@ export class NotificationsController {
   async getNotificationTemplates(
     @Query('locale') locale: string,
   ): Promise<HnippTemplate[]> {
+    this.logger.info(
+      `Fetching templates from Contentful GQL for locale: ${locale}`,
+    )
```

Comment:
fyi - this is already logged verbatim like this in notifications.service.ts, consider another message if put in the controller for other purposes

Reply:
Updated. I'm considering whether this log is even useful :thinking: